types/hapi/index.d.ts

```diff
@@ -1938,7 +1938,7 @@ export class Request extends Podium {
      * an object containing the query parameters.
      * TODO update with outcome of: https://github.com/hapijs/hapi/pull/3479
      */
-    query: Object;
+    query: any;
```

Comment:
What did you try from the README?

$ cd `DefinitelyTyped/types/hapi`
$ tsc
If tsc runs without error then you're good to go, we haven't added tslint to this yet. You'll need npm install -g typescript@2.3